f8a_report/s3_helper.py

```diff
@@ -29,23 +50,49 @@ def __init__(self):
         # S3 endpoint URL is required only for local deployments
         self.s3_endpoint_url = os.environ.get('S3_ENDPOINT_URL') or 'http://localhost'
 
-        self.s3 = boto3.resource('s3', region_name=self.region_name,
-                                 aws_access_key_id=self.aws_s3_access_key,
-                                 aws_secret_access_key=self.aws_s3_secret_access_key)
+    def s3_client(self, bucket_name):
+        """Provide s3 client for each bucket."""
+        if bucket_name == 'developer-analytics-audit-report-s3':
+            s3 = boto3.resource('s3', region_name=self.region_name,
+                                aws_access_key_id=self.aws_s3_access_key_report_bucket,
+                                aws_secret_access_key=self.aws_s3_secret_access_key_report_bucket)
+        elif bucket_name == 'hpf-pypi-insights-s3':
```

Comment:
Here bucket_name is hardcoded. Ideal scenario should be comparing this bucket name with env params.
`bucket_name == os.getenv('PYPI_MODEL_BUCKET')`. Same for all the other statements

Reply:
Done.